### DISCUSSION TITLE
Example for save TDS manually and reload from CSV

### DISCUSSION THREAD

**Discussion by jinningwang** [CURENT/andes#327] [Ideas]
Hantao,

Thanks for your development of the manually saving TDS output. I want to prepare a notebook example to demonstrate the function, will it be helpful?

Regards,
Jinning

**Comment by cuihantao** [comment]
Yes! Please check plot.py for reloading from the npz or the CSV file.

Regards,
Hantao Cui

**Comment by TheASingh** [comment]
Is it regarding saving the state of TDS to an excel file and reloading the excel file to continue the simulation where it ended?

**Reply by cuihantao** [reply]
It is not related. We don't currently support reloading from an excel file and continuing where it ended. The major issue is data consistency. Namely, the program does not/is not able to check if the data from the excel file is valid solutions to the current system and scenario. 

Loading data from excel was used to stream arbitrary data in the particular format implemented in ANDES.
